docs/en/ufs/S3.md

```diff
@@ -279,44 +249,92 @@ alluxio.underfs.s3.upload.threads.max=80
+often inconvenient to read and use in practice.
+Optionally, property
 `alluxio.underfs.s3.owner.id.to.username.mapping` can be used to specify a preset S3 canonical id to
-Alluxio username static mapping, in the format "id1=user1;id2=user2".  The AWS S3 canonical ID can
-be found at the console `https://console.aws.amazon.com/iam/home?#/security_credentials`.
-Please expand the "Account Identifiers" tab and refer to "Canonical User ID".
+Alluxio username static mapping, in the format "id1=user1;id2=user2".
+For example, edit `alluxio-site.properties` to include
+
+```properties
+alluxio.underfs.s3.owner.id.to.username.mapping=\
+79a59df900b949e55d96a1e698fbacedfd6e09d98eacf8f8d5218e7cd47ef2be=john
+```
+
+This configuration helps Alluxio recognize all objects owned by this AWS account as owned by user `john` in Alluxio namespace.
+To find out the AWS S3 canonical ID,
+check the console `https://console.aws.amazon.com/iam/home?#/security_credentials`,
+expand the "Account Identifiers" tab and refer to "Canonical User ID".
 
-### Mapping from S3 ACL to Alluxio permission
+### File Permission
 
 Alluxio checks the S3 bucket read/write ACLs to determine the owner's permissions to a Alluxio
-file. For example, if the S3 user has read-only access to the underlying bucket, the mounted
-directory and files will be set with 0500 permissions. If the S3 user has full access to the underlying bucket,
-the mounted directory and files will be set with 0700 permissions.
+file.
+For example, if the AWS user has read-only access to the mounted bucket, the mounted
+directory and files will be set with `0500` permissions.
+If the AWS user has full access to the underlying bucket,
+the mounted directory and files will be set with `0700` permissions.
+
+With `0500` or `0700` as the default file permissions, Alluxio file system users other than the
+file owner can not access the files under the moun point.
+This can lead to problems for data shared by different applications running under a different users.
+For example, a Presto job running under user `presto` may result in an error like
+`Query failed: Failed to list directory` when accessing the mount point with owner `john`.
+In  `master.log`, one can find an error like
+```
+Error=alluxio.exception.AccessControlException: Permission denied: user=presto, access=--x, path=/mnt/s3/myobject: failed at s3, inode owner=john, inode group=john, inode mode=rwx------
+```
 
-### Mount point sharing
+This is because the mounted directory has permission `0700` and thus application user `presto`
+is not able to access this file.
+To share the S3 mount point with other users in Alluxio namespace, one can choose:
 
-If you want to share the S3 mount point with other users in Alluxio namespace, you can enable
-`alluxio.underfs.object.store.mount.shared.publicly`.
+1. (Option1): Set
+`alluxio.master.mount.table.root.shared` in `alluxio-site.properties` for root mount or add `--shared` on `fs mount` command;
```

Comment:
`alluxio.master.mount.table.root.shared=??` is it a boolean to be set to `true`?